craft_application/models/project.py

```diff
@@ -311,3 +312,12 @@ def _validate_bare_base_requires_build_base(self) -> Self:
             raise ValueError(f"A build-base is required if base is {self.base!r}")
 
         return self
+
+    @classmethod
+    def _get_devel_bases(cls) -> Iterable[DevelBaseInfo]:
```

Comment:
@lengau I added this on a separate commit as a possible solution to let applications have finer control on what's 'devel'. We can drop this no problem, thought I'd add it to possibly have some discussion